app/assets/javascripts/orders.coffee

```diff
@@ -17,7 +17,7 @@ showOrderDialog = (orderId) ->
     error: (jqXHR, textStatus, errorThrown) ->
       alert "Error occurred"
 
-window.orderRowClicked = (event, row, element) ->
+orderRowClicked = (event, row, element) ->
```

Comment:
If you dropped the `window.` because it looked suspiciously unnecessary... maybe we should create a new function that would make it more obvious what is happening... perhaps in a new `expose.coffee`:
```coffeescript
expose = (name, fn) ->
  window[name] = fn

expose "expose", expose
```

Then this line would become:
```coffeescript
expose "orderRowClicked", (event, row, element) ->
  ...
```